website/docs/reference/resource-configs/fabricspark-configs.md

```diff
@@ -0,0 +1,295 @@
+---
+title: "Microsoft Fabric Spark configurations"
+id: "fabricspark-configs"
+description: "Microsoft Fabric Spark configurations - Read this in-depth guide to learn about configurations in dbt."
+---
+
+<!----
+To-do:
+- use the reference doc structure for this article/split into separate articles
+--->
+
+## Configuring tables
+
+When materializing a model as `table`, you may include several optional configs that are specific to the dbt-spark plugin, in addition to the standard [model configs](/reference/model-configs).
+
+| Option  | Description                                                                                                                        | Required?               | Example                  |
+|---------|------------------------------------------------------------------------------------------------------------------------------------|-------------------------|--------------------------|
+| file_format | The file format to use when creating tables (`delta`). | Required | `delta`|
+| location_root  | The created table uses the specified lakehouse to store its data. The table alias is appended to it.                               | Optional                | `/files/`              |
+| partition_by  | Partition the created table by the specified columns. A directory is created for each partition.                                   | Optional                | `date_day`              |
+| clustered_by  | Each partition in the created table will be split into a fixed number of buckets by the specified columns.                         | Optional               | `country_code`              |
+| buckets  | The number of buckets to create while clustering                                                                                   | Required if `clustered_by` is specified                | `8`              |
+
+## Incremental models
+
+dbt seeks to offer useful, intuitive modeling abstractions by means of its built-in configurations and <Term id="materialization">materializations</Term>. Because there is so much variance between Fabric Spark job clusters out in the world—not to mention the powerful features offered by the Delta file format and custom runtime—making sense of all the available options is an undertaking in its own right.
+
+For that reason, the dbt-fabricspark plugin leans heavily on the [`incremental_strategy` config](/docs/build/incremental-models#about-incremental_strategy). This config tells the incremental materialization how to build models in runs beyond their first. It can be set to one of three values:
```

Comment:
```suggestion
For that reason, the dbt-fabricspark plugin leans heavily on the [`incremental_strategy` config](/docs/build/incremental-strategy). This config tells the incremental materialization how to build models in runs beyond their first. It can be set to one of three values:
```